login_server/loginserver.py

```diff
@@ -330,6 +330,22 @@ def handle_http_request_message(self, msg):
                 'online_players': len(self.players),
                 'online_servers': len(self.game_servers)
             }, sort_keys=True, indent=4))
+        elif msg.env['PATH_INFO'] == '/detailed_status':
+            online_game_servers_list = [
+                {'locked':      gs.password_hash is not None,
+                 'mode':        gs.game_setting_mode,
+                 'name':        gs.description,
+                 'map_id':      gs.map_id,
```

Comment:
Do you have a conversion from the map ID to a readable name?

Reply:
I don't yet. Is there a reference for map ID values? If not I was just planing to figure them out manually.

Reply:
I don't think the login server knows right now, but perhaps we could add it there, It would be a shame to have to bother clients with it.

Reply:
Wow, that's amazing, thanks. Should I make a switch case function in loginserver.py to convert from the IDs? Or should I put this function in another file? The function could look something like:
```
match map_id:
    case 1447:
        map = "Katabatic"
        game_type = "CTF"
    case 1458:
        map = "Outskirts"
        game_type = "Rabbit"
    case 1480:
        map = "Drydock Night"
        game_type = "TDM"
    case etc...:
        etc...
```

Reply:
Something like that would be ok, but the match statement is too new. I must admit I haven't paid much attention to which version taserver requires, but I think we shouldn't use anything that requires a more recent version than 3.6.

Be sure to also have an "unknown" case.

Reply:
Oops! You're right, match-case isn't actually in python yet... it comes out in 3.10.
Alrighty, I've added this function into loginserver.py, although it is quite large. See commit ab485c2 below.

Reply:
Hmm.. better make it a dictionary.

Reply:
Yep, good like it is now.